ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir: "local_backups"
+files_to_backup:
+  etc_pulp:
+    src_dir: /etc/pulp
+    file: etc_pulp
+  etc_pki_pulp:
+    src_dir: /etc/pki/pulp
+    file: etc_pki_pulp
+  var_lib_pulp:
+    src_dir: /var/lib/pulp
+    file: var_lib_pulp
+
+db_to_bkup:
+  mongodb:
```

Comment:
Is this extra level of indirection necessary? I think that this variable could be written as:

```yaml
db_to_bkup
  dir: mongodb_backup_dir
  file: mongodb_backup
```

The references to this variable could then be simplified. Instead of this:

```yaml
- name: Archive database backups
  archive:
    path: "{{ remote_bkup_dir }}/{{ item.value.dir }}"
    dest: "{{ remote_bkup_dir }}/{{ item.value.file }}.bz2"
    format: bz2
  with_dict: "{{ db_to_bkup }}"
```

One could have this:

```yaml
- name: Archive database backup
  archive:
    path: "{{ remote_bkup_dir }}/{{ db_to_bkup.dir }}"
    dest: "{{ remote_bkup_dir }}/{{ db_to_bkup.file }}"
    format: bz2
```